netbox/extras/scripts.py

```diff
@@ -500,6 +602,61 @@ def run_script(data, job, request=None, commit=True, **kwargs):
     # Add the current request as a property of the script
     script.request = request
 
+    def signature(fn):
+        """
+        Taken from django.tables2 (https://github.com/jieter/django-tables2/blob/master/django_tables2/utils.py)
```

Comment:
We can't reuse code from other projects. Let's rewrite these functions from scratch to serve our specific needs.